src/test/java/com/jcabi/xml/XMLDocumentTest.java

```diff
@@ -239,16 +240,24 @@ public void printsWithAndWithoutXmlHeader() throws Exception {
      */
     @Test
     public void parsesInMultipleThreads() throws Exception {
-        new Runnable() {
```

Comment:
@tmjee you know what to do.
Same issues are in ALL tests:
- inline
- invokeAll
- executor shutdown management.

Please, address them all